pandas/util/testing.py

```diff
@@ -286,6 +286,14 @@ def _skip_if_mpl_1_5():
         pytest.skip("matplotlib 1.5")
 
 
+def _skip_if_mpl_1():
+    import matplotlib
```

Comment:
yes that's fine. I realize this just raises ``AttributeError``, so that would be caught by the tests anyhow.